go/signedexchange/cbor/encoder_test.go

```diff
@@ -3,6 +3,7 @@ package cbor
 import (
```

Comment:
It depends on whether you want to do black-box test or not, but if black-box test is needed here, I'd change this package name to `cbor_test` and use dot-import https://golang.org/ref/spec#Import_declarations